core/src/apps/cardano/layout.py

```diff
@@ -734,6 +739,28 @@ def _format_stake_credential(
         raise ValueError
 
 
+def _format_drep(drep: messages.CardanoDRep) -> tuple[str, str]:
+    if drep.type == CardanoDRepType.KEY_HASH:
+        assert drep.key_hash is not None  # validate_drep
+        return (
+            "Delegating to key hash:",
+            bech32.encode(bech32.HRP_KEY_HASH, drep.key_hash),
+        )
+    elif drep.type == CardanoDRepType.SCRIPT_HASH:
+        assert drep.script_hash is not None  # validate_drep
+        return (
+            "Delegating to script:",
+            bech32.encode(bech32.HRP_SCRIPT_HASH, drep.script_hash),
+        )
```

Comment:
@janmazak just to make sure - the drep key hash / script hash should be displayed with the usual `addr_vkh` and `script` bech32 prefixes, right?

Reply:
The source code of [this demo app](https://ryun1.github.io/cip95-cardano-wallet-connector/) suggests the prefix should be `drep`
```js
            const dRepIDBech32 = dRepID.to_bech32('drep');
```
But I wasn't able to find anything about it elsewhere

Reply:
> https://github.com/cardano-foundation/CIPs/blob/master/CIP-0005/README.md?plain=1#L118

@Ryun1 thanks, that explains key-hash drep IDs, is it also specified somewhere whether a script-hash drep ID should also have this prefix, or should it just be "script"?